unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts

```diff
@@ -147,7 +147,7 @@ export function DynamicFacetValueTest() {
 
     describe('when the value has the state "idle"', () => {
       it('should return the correct aria-label', () => {
```

Comment:
i would move this test outside of the `'when the value has the state "idle"` condition since there is no difference with it being idle or selected

Reply:
Agreed. Moved it outside of the 'described' scope.